src/app.rb

```diff
@@ -0,0 +1,94 @@
+require_relative './book'
+require_relative './student'
+require_relative './teacher'
+require_relative './rental'
+require_relative './modules/list'
+require_relative './modules/check'
+
+class App
+  def initialize
+    @list_people = []
+    @list_books = []
+    @rentals = []
+  end
+
+  include ShowList
+  include CheckConditions
+
+  def create_person
+    puts 'Do you want to create a Student (1) or a Teacher (2) [Input the number]:'
+    create_person_input = gets.chomp.to_i
+    case create_person_input
+    when 1
+      create_student
+    when 2
+      create_teacher
+    else
+      puts 'Invalid please choose [1] or [2]'
+      create_person
+    end
+  end
+
+  def create_student
+    puts 'Age:'
+    student_age = gets.chomp.to_i
+
+    puts 'Name:'
+    student_name = gets.chomp.capitalize
+    # label = 'Student'
+
+    puts 'Do you have parent permission? [y/n]:'
+    student_permission = gets.chomp
+    permission_checker(student_permission)
+
+    student = Student.new(student_age, student_name, student_permission)
+    @list_people.push(student)
+    puts 'Person created successfully!'
+  end
+
+  def create_teacher
+    puts 'Age:'
+    teacher_age = gets.chomp.to_ire
```

Comment:
- [ ] @team great job working on the changes requested, however, am getting this error when running the up one method'to_ire' is not recognized
![image](https://user-images.githubusercontent.com/61408860/210318801-c76cdea7-4517-44d0-b98a-f29aa147ee30.png)

Reply:
thanks @danmainah, this was a typo mistake and I have fixed it